src/native/fs/fs_napi.cpp

```diff
@@ -2115,6 +2128,32 @@ set_debug_level(const Napi::CallbackInfo& info)
     return info.Env().Undefined();
 }
 
+/**
+ * register noobaa args to GPFS
+ */
+static Napi::Value
+register_gpfs_noobaa(const Napi::CallbackInfo& info)
+{
+    DBG1("FS::register_gpfs_noobaa " << gpfs_down_delay);
+    struct gpfs_ganesha_noobaa_arg gpfs_noobaa_arg = { 
+        0, 
+        gpfs_down_delay ? std::stoi(gpfs_down_delay) : 1000,
+        0,
+    };
```

Comment:
Lets send all these from js -

```suggestion
    Napi::Object params = info[0].As<Napi::Object>();

    struct gpfs_ganesha_noobaa_arg args = {
        params["version"].ToNumber(),
        params["delay"].ToNumber(),
        params["flags"].ToNumber(),
    };
```